middleware/src/main/java/br/ufrn/dimap/middleware/infrastructure/lifecycleManagement/LifecycleManagement.java

```diff
@@ -0,0 +1,74 @@
+package br.ufrn.dimap.middleware.infrastructure.lifecycleManagement;
+
+
+import br.ufrn.dimap.middleware.infrastructure.lifecycleManagement.modelo.AbsoluteObjectReference;
+import br.ufrn.dimap.middleware.infrastructure.lifecycleManagement.modelo.ILifecycleManagement;
+
+/**
+ * The Class LifecycleManagement.
+ * 
+ * @author
+ * @version 1.0
+ * @see Lifecycle Management
+ */
+public class LifecycleManagement implements ILifecycleManagement {

```

Comment:
Change interface name to LifecycleManagementImpl, following the pattern used by the others classes and interfaces